gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -157,6 +157,10 @@ message InterceptHtlcResponse {
     Forward forward = 3;
   }
 
+  // The HTLC payment hash.
+  // Value is not guaranteed to be unique per intercepted HTLC
+  bytes payment_hash = 6;
```

Comment:
Maybe put this at the bottom so that the numbers line up, weird to see 6 out of order

Reply:
Done.